spec/requests/subscriptions_spec.rb

```diff
@@ -4,6 +4,11 @@
 RSpec.describe "Subscriptions" do
   let(:vacancy) { create(:vacancy, organisations: [build(:school)]) }
 
+  before do
+    mock_response = [double(country: "United Kingdom")]
+    allow(Geocoder).to receive(:search).and_return(mock_response)
+  end
```

Comment:
:raising_hand_man: Suggestion.

To avoid all this boilerplate on the "default" case of within UK locations, maybe this setup could be extracted to the [rails spec helper](https://github.com/DFE-Digital/teaching-vacancies/blob/0d724e4b0f82c4e9ce3c2ed46a46f46b35c7edd9/spec/rails_helper.rb#L56-L81)?

This way could be invoked on the needed tests with a RSpec tag or even be always defined by default :shrug: